### ISSUE TITLE
Trivia questions

### ISSUE THREAD

**Issue by XITACE** [SmashMC-Development/Suggestions#991]
Hi, although the trivia questions give us money, I still miss those times where you can answer trivia questions for 3 rare candies. I don’t wish the money reward to be gone but I do wish the rare candy rarity can be risen, me and most player are legit sick of spending a billion years to level up every pokemon

**Comment by morrow12** [comment]
I agree with this rare candy  eco is very expensive because of trivia rarely giving any and the rarity of raids dropping them

**Comment by jantjewieger** [comment]
There is still a chance to get rare candies from trivia. U can get either 1 or 2. I get those pretty commonly. If you aren't getting rare candies as a reward ur just getting unlucky bc they are in the possible prize pool.

**Comment by oLimooo** [comment]
I agree with increase trivia questions rewards although based on the amount of players a server averages. Rewards for getting the questions consist of 2 Rc, 1 Rc, 100$ or 150$. To even make 1000 a player would need to answer at least 7 question, amusing their getting the highest paid rolled. 14 questions for 2k, and at least on ray, a common bp would cost around 30k. It isn't enough money to do anything. In fact, warzone cost players 1000, we cannot even afford to go into warzone after 5 questions. Increasing the amount money per question, will promote players to actually do the trivia questions. 

In addition, increasing the amount of rare candy per question would make up for the fact that staff removed Ms. Eevee. Ms. Eevee when defeated would drop 6 rare candies, allowing players to level up their pokemons and have a chance to go in warzone, dungeons, gyms, and raids. Nowadays, to buy 64 rare candies, it would cost at least in ray a minimum of 60k. 

I understand there are different difficulty that changes the rewards, however only 100$ for this type of questions? 
![image](https://user-images.githubusercontent.com/82547638/167509396-034ae272-d2e3-43a2-9b26-7048edfe4e18.png)
It is unbelievable
